imap/http_caldav_sched.c

```diff
@@ -720,27 +720,40 @@ static int imip_send(const char *cal_ownerid, const char *sched_userid,
 
     /* Don't send a bogus message - check late to not allocate our own copy */
     const char *ical_str = icalcomponent_as_ical_string(sched_data->itip);
-    if (!ical_str) goto done;
+    if (!ical_str) {
+        r = IMAP_INTERNAL;
```

Comment:
Should we log something here or do you think the r of IMAP_INTERNAL will be enough?